index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+        <img src="image/Union.png" alt="" />
```

Comment:
- [ ] for semantic purposes kindly put just the  mobile hamburger inside the `nav tag` and the logo outside of it